test/unit_tests/test_metadata_table.py

```diff
@@ -85,8 +85,8 @@ def test_bad_type_name(self):
 
         #Exercise
         with self.assertRaises(Exception) as context:
-            tables = parse_metadata_file(filename, known_ddts,
-                                         self._DUMMY_RUN_ENV)
+            parse_metadata_file(filename, known_ddts,
+                                self._DUMMY_RUN_ENV)
```

Comment:
While pylint might not see the difference, I would prefer a style that uses underscore for ignored return values, e.g.,
```suggestion
            _ = parse_metadata_file(filename, known_ddts, self._DUMMY_RUN_ENV)
```